### PR TITLE
Update documentation regarding unreferenced packs

### LINKED ISSUE #3620: Possible new info regarding `pack not referenced in any index`
Output of `restic version`
--------------------------
`restic 0.12.1 compiled with go1.16.6 on linux/amd64`

How did you run restic exactly?
-------------------------------
```bash
export AWS_ACCESS_KEY_ID=...
export AWS_SECRET_ACCESS_KEY=...
export RESTIC_PASSWORD=...
/opt/restic/restic backup -r s3:https://s3.eu-central-1.wasabisys.com/... --json --quiet --one-file-system --limit-upload=20000 --exclude-caches --iexclude=tmp --iexclude=.cache --iexclude=./var/cache /

Save(<data/afa85e951d>) returned error, retrying after 552.330144ms: client.PutObject: Maximum number of server active requests exceeded
```

For checking:
```
/opt/restic/restic check -r s3:https://s3.eu-central-1.wasabisys.com/... --read-data-subset 1%

using temporary cache in /tmp/restic-check-cache-499500195
create exclusive lock for repository
load indexes
check all packs
pack cd223624: not referenced in any index
pack 57ed8770: not referenced in any index
pack 6d0233b2: not referenced in any index
...
367 additional files were found in the repo, which likely contain duplicate data.
You can run `restic prune` to correct this.check snapshots, trees and blobs
[13:04] 100.00%  121 / 121 snapshots
read 1.0% of data packs
[5:40] 100.00%  510 / 510 packs
```

What backend/server/service did you use to store the repository?
----------------------------------------------------------------
Wasabi S3

Expected behavior
-----------------

No messages regarding unreferenced packs.

Actual behavior
---------------

Messages regarding unreferenced packs shown.

Steps to reproduce the behavior
-------------------------------

See above and my additional informations below.

Do you have any idea what may have caused this?
-----------------------------------------------

The unsuccessful `PutObject` call.

Do you have an idea how to solve the issue?
-------------------------------------------

No.

Additional background info
-------------------------------

I have possibly some new info regading the "not referenced in any index" messages. The FAQ states, that the reason for them is yet unknown.

Since a few weeks, I am backing up 3 computers every night to Wasabi S3 storage. Also, every morning, I run `restic check --read-data-subset 1%`.
Every now and then, I got a bunch of errors during backup regarding `PutObject`, but they where non-fatal as they always said "retrying after ...". (I'm not 100% sure, because I did not use `--quiet` except for the last few days, so maybe I missed a fatal error.) The checks in the morning always ran without any error or warning.
All Backup jobs run in sequence not in parallel.

Last night I had two new errors during backup which I did not experience before:
- On my linux machine, I got a fatal error from `PutObject`: `Save(<data/afa85e951d>) returned error, retrying after 552.330144ms: client.PutObject: Maximum number of server active requests exceeded`
- On a windows machine, I got a few of those:
  ```json
  {"during": "archival",
   "error": {
       "Err": 21,
       "Op": "read",
       "Path": "\\\\?\\GLOBALROOT\\Device\\HarddiskVolumeShadowCopy12\\data\\info\\abc"},
   "item": "c:\\data\\info\\abc",
   "message_type": "error"
  ```
  Maybe those are because of a Veeam Agent Backup running in parallel, which also creates a Volume Shadow Copy. I now have changed the time of the Veeam Backup, so this should not happen again. However, those backups have been running in parallel for weeks now and this is the first time I got those errors.
  
Then, in the morning, the daily `restic check` gave me 367 messages like `pack 8d6f0843: not referenced in any index`. This is the first time I see those messages. I think it is very likely that those messages a caused by either the fatal `PutObject` call or the read errors. (I suspect the former.)

Just for testing if it relates to the local cache, I ran `restic check` on the third box (another linux machine). It also outputs the warnings. I checked a few random IDs and it seems that the list is the same.

For now, I did not run `restic prune`. If any dev wants to try something out, I will gladly assist.

I guess, the fatal upload error should be healed with another backup run? Is this correct?

Did restic help you today? Did it make you happy in any way?
------------------------------------------------------------

Restic still makes me sleep much better...

### PR DESCRIPTION
**Pull request by fkusche**
Also removes an unnecessary space at the end of the last line.

What does this PR change? What problem does it solve?
-----------------------------------------------------
Update the documentation. The docs still say that the cause for the messages is unknown, which is not correct anymore.

Was the change previously discussed in an issue or on the forum?
----------------------------------------------------------------
Closes #3620

Checklist
---------

- [x] I have read the [contribution guidelines](https://github.com/restic/restic/blob/master/CONTRIBUTING.md#providing-patches).
- [x] I have [enabled maintainer edits](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork).
- [ ] I have added tests for all code changes.
- [x] I have added documentation for relevant changes (in the manual).
- [ ] There's a new file in `changelog/unreleased/` that describes the changes for our users (see [template](https://github.com/restic/restic/blob/master/changelog/TEMPLATE)).
- [ ] I have run `gofmt` on the code in all commits.
- [x] All commit messages are formatted in the same style as [the other commits in the repo](https://github.com/restic/restic/blob/master/CONTRIBUTING.md#git-commits).
- [x] I'm done! This pull request is ready for review.

### CODE REVIEW

File: doc/faq.rst [hunk 1]
```diff
@@ -49,9 +49,9 @@ looks like this:
     Fatal: repository contains errors
```
Comment (MichaelEischer) [inline doc/faq.rst:49]: The error message here is also outdated. The current variant is:
```
using temporary cache in /tmp/restic-check-cache-168381449
repository 86923eb5 opened successfully, password is correct
created new cache in /tmp/restic-check-cache-168381449
create exclusive lock for repository
load indexes
check all packs
pack 8f5c55d37d28a86cb9a20adb030d115f07bbecf05f3506998d0254f2b39b2496: not referenced in any index
pack c02e2919d799891c5ca752af8829ca1401b0a0cf135ed8b996d0e7929f8a6455: not referenced in any index
pack a9047d2ccabf343754a0108e49aba885bbe4c18277ee38b757ab7764aa721fb0: not referenced in any index
pack eeb0794fab182cbac3a1fa5bd61446782009445f98f362eb8185e587f4257125: not referenced in any index
pack bf34a6d44cceff21c29c4da44cf10245b7aa9ed1698551ce915d5d687d10c057: not referenced in any index
pack a07d4572848962f46269628cb32b1f8008d384b0028b9a64cc4487bf71dca853: not referenced in any index
pack d40ad98368d52b75b785abd8c7dc6642428f8e1815c21486934a34f9c5729f88: not referenced in any index
7 additional files were found in the repo, which likely contain duplicate data.
You can run `restic prune` to correct this.
check snapshots, trees and blobs
[0:00] 100.00%  16 / 16 snapshots
no errors were found
```

My suggestion would be to cut that down to
```
create exclusive lock for repository
load indexes
check all packs
pack 819a9a52e4f51230afa89aefbf90df37fb70996337ae57e6f7a822959206a85e: not referenced in any index
pack de299e69fb075354a3775b6b045d152387201f1cdc229c31d1caa34c3b340141: not referenced in any index
2 additional files were found in the repo, which likely contain duplicate data.
You can run `restic prune` to correct this.
check snapshots, trees and blobs
[0:00] 100.00%  16 / 16 snapshots
no errors were found
```

Comment (fkusche) [reply doc/faq.rst]: I updated my branch with your proposal.

However, the current variant in `master` is already a bit different to the current variant you describe above...?

Comment (MichaelEischer) [reply doc/faq.rst]: I'm not sure which reference you're referring to? The first log snippet is the output I get when using the latest master branch. The difference to my suggestion is that it removes the first three lines, which only add noise to the example.

Comment (fkusche) [reply doc/faq.rst]: Ah ok, then I misunderstood. I thought you where referring to the current text in `faq.rst`.

However, I took the contents of the second snippet and inserted it into `faq.rst` in my branch. That should be what you suggested, isn't it?

Comment (MichaelEischer) [reply doc/faq.rst]: It is. I was just confused by the reference to the current variant.

**Review by MichaelEischer — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**